src/components/spinner/spinner.jsx

```diff
@@ -63,7 +63,7 @@ function Spinner({
   sheet, // eslint-disable-line react/prop-types
   ...rest
 }) {
-  const usedColor = color || theme.palette.variant.primary;
+  const usedColor = color || '#0046FF';
```

Comment:
So here we're changing from a palette color to a hard coded one? I'm just gonna stop commenting on these and assume this is fine.

Reply:
good catch, should be CTA, fixing